components/projectsView/ProjectFilters.js

```diff
@@ -76,16 +74,20 @@ export default function ProjectFilters({
     <Row className="text-center">
       {isTabletOrMobile ? (
         <Col key="projectStatusFilter">
-          <Row className={`bg-light h-100 `}>
+          <Row
+            className={`h-100`}
+            style={{
+              backgroundColor: "#f8f9fa",
```

Comment:
this `bg-light` doesn't work anymore with the upgrade so I had to add the light grey background with styling